src/popup/router/components/Overview.vue

```diff
@@ -177,7 +177,7 @@ export default {
       line-height: 16px;
       color: variables.$color-white;
 
-      &:hover .truncate-mid {
+      &:hover .truncate {
```

Comment:
| develop | this PR |
| ------ | ----- |
| ![Screenshot 2021-06-02 at 17 41 25](https://user-images.githubusercontent.com/482351/120506446-98d97700-c3ce-11eb-930e-30e404ca0cea.png) | ![Screenshot 2021-06-02 at 17 40 45](https://user-images.githubusercontent.com/482351/120505760-f3be9e80-c3cd-11eb-9ba6-6777a2837338.png) |

Suggestion how to fix:

```
    .recipient {
      width: 148px;
      text-align: right;

      .truncate ::v-deep span {
        width: 100%;
      }

      .name.aens {
        padding-right: 8px;
      }
    }

    .name {
      display: block;
      margin-bottom: 4px;
      font-size: 15px;
      line-height: 16px;
      color: variables.$color-white;
      text-decoration: none;

      &:hover {
        text-decoration: underline;
      }
    }
```